README.md

```diff
@@ -5,52 +5,68 @@
 
 ## Quickstart
 
-Before you can begin with Yari, you need [Content](https://github.com/mdn/content).
-See its README which basically, says something like this:
+Before you can start working with Yari, you need a copy of the MDN [Content](https://github.com/mdn/content)
+directory — this is the content Yari will render when we run it locally. The
+easiest way to get a local copy of this is to run the following command
+(although bear in mind that you might want to clone your own fork of the repo
+instead, for when you create pull requests against the content):
 
     git clone https://github.com/mdn/content.git mdn/content
 
-Now, you just need to note where that folder is before you can start Yari.
+Take a note of where you've clone that repo before moving on.
 
 To run Yari locally, you'll first need to install [git](https://git-scm.com/),
-[Node.js](https://nodejs.org) (>= 12.0.0) and
+[Node.js](https://nodejs.org) (>= 12.0.0), and
 [Yarn 1](https://classic.yarnpkg.com/en/docs/install).
-After that, run these commands in your bash:
+
+After that, run these commands in your terminal, bearing in mind that you'll
+probably want to fork the repo first and do the `git clone` with _your_ fork
+instead of the `mdn` one:
 
     git clone https://github.com/mdn/yari.git
     cd yari
-    yarn
+
+Next run the `yarn` command to fetch all the dependencies and install the app
+locally:
+
+Now run the following command to create a `.env` file inside your `yari` repo
+root and set the `CONTENT_ROOT` environment variable equal to the path to the
+`content` repo. This is so the Yari app can find the content it needs to render.
+You'll need to replace `/path/to/mdn/content/files` with the path to the
+`/files` folder inside your clone of the `content` repo:
+
     echo CONTENT_ROOT=/path/to/mdn/content/files >> .env
+
+At this point, you can get started. Run the following lines to compile required
+files, start the Yari web server running, and open it in your browser:
```

Comment:
```suggestion
files, start the Yari [web server](https://nodejs.org/en/docs/guides/getting-started-guide/) running, and open it in your browser:
```

Reply:
Disagree with this link - people know what a web server is. I'd agree if this was a link to yari-specific web server information explaining what is special about it.

Reply:
Yup; agree with @hamishwillee